src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,25 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
+        String login = "bob";
         try {
             user = authenticationService.login("bob", "1234");
```

Comment:
The hardcoded login string "bob" should be replaced with the variable 'login' defined above to avoid duplication and potential errors in the future.